conanfile.py

```diff
@@ -26,8 +26,8 @@ class DiceHashConan(ConanFile):
         "with_test_deps": False,
         "with_sodium": False
     }
-    exports = "LICENSE"
-    exports_sources = "include/*", "CMakeLists.txt", "cmake/*", "LICENSE"
+    exports = "LICENSE-MIT", "LICENSE-APACHE"
```

Comment:
I don't think this is necessary, right? (at least if I understand the docs correctly: https://docs.conan.io/2/reference/conanfile/attributes.html#exports)